changelog.md

```diff
@@ -1,3 +1,534 @@
+* smoke: replace --invert-match cli option with --ignore-exclusions ([#14332](https://github.com/GoogleChrome/lighthouse/pull/14332))
+* smoke: support a per-runner test exclusion list ([#14316](https://github.com/GoogleChrome/lighthouse/pull/14316))
+* source-maps: throw explicit error when map is missing required fields ([#14060](https://github.com/GoogleChrome/lighthouse/pull/14060))
+* target-manager: only listen to LH sessions ([#14385](https://github.com/GoogleChrome/lighthouse/pull/14385))
+* types: use union of `puppeteer` and `puppeteer-core` ([#14435](https://github.com/GoogleChrome/lighthouse/pull/14435))
+* user-flow: update UIString comments ([#14458](https://github.com/GoogleChrome/lighthouse/pull/14458))
+* user-flow: add base flags option ([#14459](https://github.com/GoogleChrome/lighthouse/pull/14459))
+* user-flow: cleanup types ([#14442](https://github.com/GoogleChrome/lighthouse/pull/14442))
+* user-flow: i18n default names ([#14455](https://github.com/GoogleChrome/lighthouse/pull/14455))
+* user-flow: start/end navigation functions ([#13966](https://github.com/GoogleChrome/lighthouse/pull/13966))
+* uses-responsive-images: higher threshold with breakpoints ([#13853](https://github.com/GoogleChrome/lighthouse/pull/13853))
+* viewport: fix tap delay link ([#14460](https://github.com/GoogleChrome/lighthouse/pull/14460))
+* page-functions: remove all `*String` exports ([#14374](https://github.com/GoogleChrome/lighthouse/pull/14374))
+* internalize resolved configs ([#14589](https://github.com/GoogleChrome/lighthouse/pull/14589))
+* asset-saver: save flow artifacts in separate files ([#14599](https://github.com/GoogleChrome/lighthouse/pull/14599))
+* replace `Page.getResourceTree` with `Page.getFrameTree` ([#14663](https://github.com/GoogleChrome/lighthouse/pull/14663))
+* js-usage: ignore __puppeteer_evaluation_script__ ([#13952](https://github.com/GoogleChrome/lighthouse/pull/13952))
+* use main-frame LCP trace element ([#14760](https://github.com/GoogleChrome/lighthouse/pull/14760))
+* full-page-screenshot: get screenshot, nodes concurrently ([#14763](https://github.com/GoogleChrome/lighthouse/pull/14763))
+* config: prevent custom gatherer interference ([#14756](https://github.com/GoogleChrome/lighthouse/pull/14756))
+* valid-source-maps: validate url in first-party check ([#14758](https://github.com/GoogleChrome/lighthouse/pull/14758))
+* disconnect Puppeteer when started by Lighthouse ([#14770](https://github.com/GoogleChrome/lighthouse/pull/14770))
+* use `resolvedConfig` to name every resolved config ([#14600](https://github.com/GoogleChrome/lighthouse/pull/14600))
+* rename resolved config types ([#14647](https://github.com/GoogleChrome/lighthouse/pull/14647))
+* remove trace-of-tab references ([#14590](https://github.com/GoogleChrome/lighthouse/pull/14590))
+* disable bf-cache in lr/psi ([#14774](https://github.com/GoogleChrome/lighthouse/pull/14774))
+
+## ⛏️👷 Fraggle Rock
```

Comment:
Perhaps it's time to change this to a "User Flows" heading. We don't use #11313 for much anymore.